### PR TITLE
Make line width equal to point's stroke width

### PR DESCRIPTION
**Pull request by OLarionova-HORIS**
… pointrange and lollipop.

### CODE REVIEW

File: plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt [hunk 1]
```diff
@@ -67,7 +67,7 @@ internal class DefaultGeomTheme private constructor(
             var fill = BASE.fill
```
Comment (alshan) [inline plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt:67]: Let's get rid of the `BASE` field.

File: plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/aes/AestheticsUtil.kt [hunk 2]
```diff
@@ -50,35 +50,11 @@ object AestheticsUtil {
...
-//    fun sizeFromCircleDiameter(diameter: Double): Double {
-//        // px -> aes Units
-//        return diameter / 2.2
-//    }
-//
-    fun textSize(p: DataPointAesthetics): Double {
-        // aes Units -> px
-        return p.size()!! * 2
-    }
+    fun pointStrokeWidth(p: DataPointAesthetics) = AesScaling.pointStrokeWidth(p)
```
Comment (alshan) [inline plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/aes/AestheticsUtil.kt:55]: Why do we need `pointStrokeWidth()` when we already have `strokeWidth()` ?

File: plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt [hunk 3]
```diff
@@ -108,19 +108,26 @@ internal class DefaultGeomTheme private constructor(
                     size = 0.2
                 }
 
+                GeomKind.POINT,
+                GeomKind.JITTER,
+                GeomKind.Q_Q,
+                GeomKind.Q_Q_2 -> {
+                    color = inheritedColors.lineColor()
+                    fill = Colors.withOpacity(inheritedColors.lineColor(), 0.1)
```
Comment (alshan) [inline plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt:116]: We don't want to use colors with opacity.

File: plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt [hunk 4]
```diff
@@ -108,19 +108,26 @@ internal class DefaultGeomTheme private constructor(
...
-                GeomKind.Q_Q,
-                GeomKind.Q_Q_2,
                 GeomKind.LOLLIPOP -> {
                     color = inheritedColors.lineColor()
                     fill = Colors.withOpacity(inheritedColors.lineColor(), 0.1)
                     size = 2.0
+                    lineWidth = 1.0 // line width and stroke for point
                 }
 
                 GeomKind.SMOOTH -> {
```
Comment (alshan) [inline plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt:133]: This is bad: `alpha = 1.5`

---
```
                GeomKind.SMOOTH -> {
                    color = Color.MAGENTA

```
We don't want to use constant values no longer. Should be a field in `BASE` (but still get rid of the `BASE`).

**Review by alshan — CHANGES_REQUESTED** [review]

File: plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt (hunk 3 above)
Comment (OLarionova-HORIS) [reply plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt]: Changed to `fill` = <background color>

File: plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt [hunk 5]
```diff
@@ -105,28 +106,40 @@ internal class DefaultGeomTheme private constructor(
                     color = inheritedColors.lineColor()
                     fill = inheritedColors.lineColor()
                     alpha = 0.1
-                    size = 0.2
+                    size = 0.24
```
Comment (alshan) [inline plot-builder-portable/src/commonMain/kotlin/jetbrains/datalore/plot/builder/defaultTheme/DefaultGeomTheme.kt:109]: Extract the experimental multiplier (1.2) to a `val` .

**Review by alshan — CHANGES_REQUESTED** [review]

**Pull request merged**